packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -28,6 +31,7 @@ systemd:
             Environment="ETCD_PEER_CLIENT_CERT_AUTH=true"
             Environment="RKT_RUN_ARGS=${etcd_arch_rkt_args}"
             Environment="${etcd_arch_options}"
+            ExecStartPre=/opt/etcd-private-interface
```

Comment:
`opt` seems to be collecting random files (bootkube files, persist-data script, now this too). Would it make sense to create a folder there to store this, and avoid a collection of random files? Not sure, just thinking outloud

Reply:
It is removed now.